### PR TITLE
using default USD currency in admin/sales location

### LINKED ISSUE #2402: Admin/sales statistics not being rendered
**Describe the bug**

**To Reproduce**
1. Go to Admin/Sales
2. Click on Locations
3. See error

**Expected behavior**

The non rendered stats should display a zero instead

**Screenshots**

image

**Desktop (please complete the following information):**
 - OS: [e.g. iOS] Ubuntu
 - Browser [e.g. chrome, safari] Chrome

**Additional context**

Working on it

### PR DESCRIPTION
**Pull request by uds5501**
#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The acceptance, integration, unit tests and linter pass locally with my changes 
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

#### Short description of what this resolves:
We can't use paymentCurrency of an event based on location because on a location, different events with different  currencies maybe present and they can't be displayed under a single tab using a single payment Currency unless standardised.

#### Changes proposed in this pull request:

- Use USD as default rendering currency

Fixes #2402

### CODE REVIEW

File: app/templates/admin/sales/locations.hbs [hunk 1]
```diff
@@ -26,11 +26,11 @@
             <tr>
               <td>{{entry.locationName}}</td>
               <td class="right aligned">{{entry.sales.completed.ticket_count}}</td>
-              <td class="right aligned">{{currency-symbol entry.paymentCurrency}} {{number-format entry.sales.completed.sales_total}}</td>
+              <td class="right aligned">US$ {{number-format entry.sales.completed.sales_total}}</td>
```
Comment (kushthedude) [inline app/templates/admin/sales/locations.hbs]: This is not the valid fix @uds5501 as the line {{currency-symbol entry.paymentCurrency}} was written to correctly render the currency parser and by setting it to default 'US $' , You are not making valid fix.

Comment (uds5501) [reply app/templates/admin/sales/locations.hbs]: @kushthedude It actually is a valid fix . Did you read the pertaining argument in the issue itself?
The location part of the admin/sales  is not yet ready for event based currency parsing.

At a location `ABC` suppose there are 3 events. 2 of them use `USD` and one of them uses `CAD`. What would you show then, `USD`\`CAD`?  :)

Comment (kushthedude) [reply app/templates/admin/sales/locations.hbs]: Thats what i was saying , How did you came to know location part cant render event bases currency parsing?

Comment (kushthedude) [reply app/templates/admin/sales/locations.hbs]: How can i make them ready for currency parsing? 
@uds5501

Comment (uds5501) [reply app/templates/admin/sales/locations.hbs]: @kushthedude Regarding that, I have proposed #2253 to be worked during GSoC period. I will standardise the amount of other currency into USD and that should do it!

**Comment by kushthedude** [comment]
@CosmicCoder96 shouldnt this supposed to be in #2376

**Pull request merged**